src/styles/_catalog.scss

```diff
@@ -14,6 +16,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition_duration;
```

Comment:
set transition to specific rules not for everything

Reply:
Didn't fixed

Reply:
?????